shared/teams/really-leave-team/container-chat.js

```diff
@@ -0,0 +1,39 @@
+// @flow
```

Comment:
Marco suggested I copy this - it is the same as container.js except for onLeave navigation